app/components/PhysicsPlane.js

```diff
@@ -0,0 +1,42 @@
+import {
+  MeshPhongMaterial
+} from 'three';
+
+import {
+  PlaneModule
+} from 'physics-module-ammonext';
+
+// Meshes
+import {
+  Plane
+} from '@whs+meshes';
+
+import {MeshComponent} from '@whs/core/MeshComponent';
+
+export class PhysicsPlane extends Plane {
```

Comment:
Is there a need for `PhysicsPlane`, `PhysicsSphere` ... ?
I mean that they are used only **once** in code.